spec/lib/task-data/tasks/create-raid-spec.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe(require('path').basename(__filename), function () {
+    var base = require('./base-tasks-spec');
+    //var hogan = require('hogan.js');
+
+    base.before(function (context) {
+        context.taskdefinition = helper.require('/lib/task-data/tasks/create-raid.js');
```

Comment:
Same naming comment, but for the filename, perhaps `create-megaraid.js` or something?